### PR TITLE
namespace refactor; fixing memory leak

### PR DESCRIPTION
**Pull request by jiyinyiyong**
memory leak because components that stored into `*store` contains closures, and closures also contains previous version of `@*store` data, and the reference keeps on, data in store never really got GCed,
image
image

after fix, components and elements(mainly for closures) are not stored into `*store`, but using a local dictionary instead.

### CODE REVIEW

**Comment by jiyinyiyong** [comment]
https://user-images.githubusercontent.com/449224/121784370-49712300-cbe6-11eb-8af7-79b8b855de18.mov

**Comment by jiyinyiyong** [comment]
https://user-images.githubusercontent.com/449224/121786521-e46ffa00-cbf2-11eb-8e2b-62027a928687.mov

File: main.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
-import { main_BANG_ } from "./js-out/quamolit.main.js"
+import { main_$x_ } from "./js-out/quamolit.app.main.js"
```
Comment (zzz6519003) [inline main.js:2]: main_$x_ ?

Comment (jiyinyiyong) [reply main.js:2]: keep in mind that calcit-js is a Lisp dialect and in Lisp, it's possible to have `main!` as a function name, according to change in https://github.com/calcit-lang/calcit_runner.rs/pull/56 we have `!` compiled to `_$x_` and now we got this.

**Pull request merged**